commands/key_add.go

```diff
@@ -0,0 +1,89 @@
+package commands
+
+import (
+	"fmt"
+
+	"github.com/MichaelMure/git-bug/cache"
+	"github.com/MichaelMure/git-bug/commands/select"
+	"github.com/MichaelMure/git-bug/identity"
+	"github.com/MichaelMure/git-bug/input"
+	"github.com/MichaelMure/git-bug/util/interrupt"
+	"github.com/spf13/cobra"
+)
+
+var (
+	keyAddArmoredFile    string
+	keyAddArmored        string
+	keyAddExpireUnixTime int64
+)
+
+func runKeyAdd(cmd *cobra.Command, args []string) error {
+	backend, err := cache.NewRepoCache(repo)
+	if err != nil {
+		return err
+	}
+	defer backend.Close()
+	interrupt.RegisterCleaner(backend.Close)
+
+	id, args, err := _select.ResolveUser(backend, args)
+	if err != nil {
+		return err
+	}
+
+	if len(args) > 0 {
+		return fmt.Errorf("unexpected arguments: %s", args)
+	}
+
+	if keyAddArmoredFile != "" && keyAddArmored == "" {
+		keyAddArmored, err = input.BugCommentFileInput(keyAddArmoredFile)
```

Comment:
I suppose that works, but please make a properly named version of this function, if only for the semantic. Maybe a really generic one like `input.FileInput(fileName string)` ? I suppose you don't need the post-process step as well.